sonar-java-plugin/src/main/java/org/sonar/plugins/java/bridges/DesignBridge.java

```diff
@@ -66,7 +66,7 @@ public DesignBridge(SensorContext context, DirectedGraph<Resource, Dependency> g
 
   public void saveDesign(Project sonarProject) {
     Collection<Resource> directories = resourceMapping.directories();
-    TimeProfiler profiler = new TimeProfiler(LOG).start("Package design analysis");
+    Profiler profiler = Profiler.create(LOG).startInfo("Package design analysis");
     LOG.debug("{} packages to analyze", directories.size());
 
     IncrementalCyclesAndFESSolver<Resource> cyclesAndFESSolver = new IncrementalCyclesAndFESSolver<>(graph, directories);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Resource"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)